app/graphql/types/payment_provider_customers/provider_input.rb

```diff
@@ -7,6 +7,7 @@ class ProviderInput < BaseInputObject
 
       argument :provider_customer_id, ID, required: false
       argument :sync_with_provider, Boolean, required: false
+      argument :provider_payment_methods, [String], required: true
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `provider_payment_methods` should appear before `sync_with_provider`.